composer.json

```diff
@@ -44,5 +44,9 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
+        "test": "phpunit",
+        "php-coveralls": "php-coveralls"
     }
```

Comment:
Added the scripts section so that the scripts in `vendor/bin` can be executed on Windows as well by using the `composer run-script` feature.